website/docs/metrics.md

```diff
@@ -1,11 +1,87 @@
 ---
 id: metrics
-title: Metrics
+title: Metrics & Observability
 ---
+## Observability
+
+We want to enable operators to glean information about how Gatekeeper is performing, as a system. To that end, operators can choose to enable execution stats that are gathered when Constraints are evaluated in the engine. At present these stats can only be logged.
```

Comment:
No need to inject our persona ("we want") into functionality docs. Better to focus on use cases. I.e.:

"This section covers how to gather more detailed statistics about Gatekeeper's query performance. This can be helpful diagnosing situations such as identifying a constraint template with a long execution time. Statistics are written to Gatekeeper's stdout logs."

Also, no need to hint at functionality ("at present these stats can only be logged"). This doc is about what users can currently do. This also implies that stdout logs are insufficient. We shouldn't prime readers for disappointment.

Reply:
Question: Is this not a metric because of cardinality concerns?

Reply:
Yep. We have aggregated metrics in the form of audit runtime and webhook review time.

Metrics would be especially hard here b/c different engines will have different granularity of stats. For instance, Rego can only report on the runtime for all constraints for a given template, not each individual constraint's runtime. This is for performance reasons. Supporting K8s's Validating Admission Policy would have no such limitation.

Reply:
tl;dr yes. we went w logging for emission in this case. We can revisit the emission mechanism (logging, metrics, aggregated or not) down the road and the consequences.